pkg/services/ldap/settings.go

```diff
@@ -15,8 +15,18 @@ import (
 
 const defaultTimeout = 10
 
-// Config holds list of connections to LDAP
+// Config holds parameters from the .ini config file
 type Config struct {
+	Enabled           bool
+	ConfigFilePath    string
+	AllowSignUp       bool
+	SkipOrgRoleSync   bool
+	SyncCron          string
+	ActiveSyncEnabled bool
+}
```

Comment:
Thanks, I'll look into this.

Reply:
One more thing to consider here: the LDAP config can be modified at runtime via the SSO settings service. If we include the LDAP config into the main config as you suggested then the Reload function will modify the main config (only the LDAP part) whenever the LDAP settings are updated from SSO. Mutating the config at runtime will probably need some locking mechanism. AFAIK the config is populated at startup and never modified at runtime.

That's why I wanted to work with a copy of the LDAP settings instead of the entire config. What do you think?

Reply:
Yeah that makes sense